src/Components/CreateImageWizard/formComponents/TargetEnvironment.js

```diff
@@ -237,6 +247,18 @@ const TargetEnvironment = ({ label, isRequired, ...props }) => {
           name="Bare metal installer"
           data-testid="checkbox-image-installer"
         />
+        {[RHEL_8, CENTOS_8].includes(getState()?.values?.release) &&
+          isBeta() && (
+            <Checkbox
+              label="Windows Subsystem for Linux (WSL)"
```

Comment:
Should probably also have extension in the parentheses.

```suggestion
              label="WSL - Windows Subsystem for Linux (.tar.gz)"
```